### DISCUSSION TITLE
point_0812

### DISCUSSION THREAD

**Discussion by andrea29-star** [cloudsen12/dataset#503] [Roy]
"20200712T105621_20200712T110348_T30UXB"
![image](https://user-images.githubusercontent.com/54947195/148126598-cdfe59f0-fc16-4d1c-a487-7567858c508b.png)

"20200607T105619_20200607T105621_T30UXB"
![image](https://user-images.githubusercontent.com/54947195/148126905-0fdfe6b7-940e-4d51-b422-8a2c085d1523.png)
![image](https://user-images.githubusercontent.com/54947195/148126982-356f8649-2e96-488f-ac25-229319822f0d.png)

"20200513T105621_20200513T110348_T30UXB"
![image](https://user-images.githubusercontent.com/54947195/148127114-4cab488b-16a7-484a-9717-fa3f77b55e99.png)

"20190701T110621_20190701T111040_T30UXB"
![image](https://user-images.githubusercontent.com/54947195/148127282-99b6845d-7964-4915-b4e0-459f1c872bb1.png)

@ryali93

**Comment by AntoniCesar** [comment]
La primera imagen corresponde al codigo "20200712T105621_20200712T110348_T30UXB" y no el "20200712T105621_20200712T110348_T30UXB"
![Captura](https://user-images.githubusercontent.com/86502862/154706786-4dadf1c0-bb12-4292-a522-02ce59ee1a48.JPG)

**Comment by AntoniCesar** [comment]
"20200607T105619_20200607T105621_T30UXB"
![Captura](https://user-images.githubusercontent.com/86502862/154711134-d893ed61-b2ac-4e3a-b3b8-1c7949be3b8f.JPG)
